io.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
```

Comment:
We need to add this file to .gitignore of the main code so that it is not included when we merge it. 

Probably we can call it a common name for all users like "testcode*.py".